stdlib/public/runtime/CMakeLists.txt

```diff
@@ -213,7 +213,7 @@ add_swift_target_library(swiftImageRegistrationObjectCOFF
                   SWIFT_COMPILE_FLAGS ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
                   INSTALL_IN_COMPONENT none)
 
-foreach(sdk ${SWIFT_CONFIGURED_SDKS})
+foreach(sdk ${SWIFT_SDKS})
```

Comment:
This change seems principally incorrect: only the targets that have been setup to be built are the ones that we should be setting up targets for.

Reply:
No, [as explained when I added this change last month](https://github.com/apple/swift/pull/30170#issuecomment-607065583), `SWIFT_SDKS` is the right one because it is possible for the developer running `build-script` to ask for only a subset of the configured SDKs to be built with `--stdlib-deployment-targets`, which is where `SWIFT_SDKS` comes from. It is why `SWIFT_SDKS` is used everywhere else in this repo, with [`SWIFT_CONFIGURED_SDKS` now only relegated to making sure the passed-in `SWIFT_SDKS` is a subset of it](https://github.com/apple/swift/blob/2b2b5b8f081fcd2f2f0a3609276f920f96c1f7b4/CMakeLists.txt#L797).